### ISSUE TITLE
Not able to see details of Revocation Transaction

### ISSUE THREAD

**Issue by cryptoBeliever** [symbol/explorer#1120]
## What's the issue
Currently, the explorer is not able to present Revocation Transaction correctly. It shows that transaction doesn't exist.
It worked before so looks like a regression.

For the following URL https://testnet.symbol.fyi/transactions/0E5044ED05D4380FF5A240A80DEFFEDD0240C6B3723896CD366692C47989D55B in the Chrome console there are visible error:

```
DataSet.js:53 TypeError: a.toHex is not a function
    at CreateTransaction.js:127:58
    at f (regeneratorRuntime.js:86:17)
    at Generator._invoke (regeneratorRuntime.js:66:24)
    at Generator.next (regeneratorRuntime.js:117:21)
    at r (asyncToGenerator.js:3:20)
    at s (asyncToGenerator.js:25:9)
```

### Screenshot:

![image](https://user-images.githubusercontent.com/17684281/193120768-416f5455-2c5a-4297-901d-7dac1bbf58bc.png)

## What's the expected behavior

Present Revocation Transaction details

**Comment by AnthonyLaw** [comment]
it should fixed in 9a758ad9017f4b3043b3476044ca1944b8812fd9

Try this to verify

go to `/transactions` and select `mosaic` in dropdown box, and search for `Reclaim` type

https://explorer-dev.netlify.app/

**Comment by cryptoBeliever** [comment]
Fixed in dev.
